cli/crates/cli/src/start/configuration.rs

```diff
@@ -492,4 +499,24 @@ mod tests {
         invalid value: integer `1000000000000000000`, expected u16
         "###);
     }
+
+    #[test]
+    fn telemetry() {
```

Comment:
Maybe we need a few more tests for this?

Reply:
hm, there are [unit tests](https://github.com/grafbase/grafbase/pull/1407/files#diff-7596b966e4a4dab502426e39482aa121c31b2e1e667e5f141cdd98dccbec9cafR397) and [integration-tests](https://github.com/grafbase/grafbase/pull/1407/files#diff-08894959da040b5465cb8a63a4a0f4efc1144bacfb04b58ac961f42685d61274R1). Anything else you'd like to see?